### PR TITLE
Updated example playbooks to reflect module updates

### PR DESCRIPTION
**Pull request by stalabi1**
##### SUMMARY

I updated the example playbooks to reflect module updates.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

sonic_bgp_as_paths
bgp_l3_fabric
common_examples

### CODE REVIEW

**Comment by kerry-meyer** [comment]
There is an example of "bfd: true" that needs to be updated in the "/playbooks/bgp_l3_fabric/group_vars/leaf.yaml" file.

File: playbooks/common_examples/sonic_bgp.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@
         commands: 
           - route-map aa permit 11
           - route-map bb permit 22
+          - ip prefix-list p1 seq 1 permit 1.1.1.1/1
```
[the remark this answers is not in the document]
Comment (stalabi1) [reply playbooks/common_examples/sonic_bgp.yaml]: Done

File: playbooks/common_examples/sonic_bgp.yaml [hunk 2]
```diff
@@ -146,14 +148,31 @@
...
                     extended_nexthop: true
+                  address_family:
+                    afis:
+                      - afi: ipv4
+                        safi: unicast
+                        ip_afi:
+                          default_policy_name: aa
+                          send_default_route: false
+                        prefix_limit:
+                          max_prefixes: 2
```
[the remark this answers is not in the document]
Comment (stalabi1) [reply playbooks/common_examples/sonic_bgp.yaml]: Done

File: playbooks/common_examples/sonic_bgp.yaml [hunk 3]
```diff
@@ -231,6 +254,15 @@
                         direction: in
                     route_reflector_client: true
                     route_server_client: true
+                    ip_afi:
+                      default_policy_name: bb
+                      send_default_route: true
+                    prefix_limit:
+                      max_prefixes: 100
+                      prevent_teardown: true
+                      warning_threshold: 80
```
Comment (kerry-meyer) [inline playbooks/common_examples/sonic_bgp.yaml:263]: It would be good to also show "restart_timer" here.

**Comment by kerry-meyer** [comment]
Although the sonic_bgp_as_paths "module" file has been updated to include the "permit" attribute as one of the "options", the "permit" attribute should also be added to the set of example playbook cases in that file (for the "merged" and "deleted" example cases).

**Comment by kerry-meyer** [comment]
I have finished checking all of the changes for this change set and have no additional comments.

File: playbooks/common_examples/sonic_bgp.yaml (hunk 3 above)
Comment (stalabi1) [reply playbooks/common_examples/sonic_bgp.yaml:263]: prevent_teardown and restart_timer cannot be enabled at the same time, there are other tasks in the playbook that show prevent_teardown as false and specify a restart_timer

**Comment by stalabi1** [comment]
> Although the sonic_bgp_as_paths "module" file has been updated to include the "permit" attribute as one of the "options", the "permit" attribute should also be added to the set of example playbook cases in that file (for the "merged" and "deleted" example cases).

The permit attribute already existed in the module file as an attribute option. The changes made are only to the "merged" and "deleted" example cases. I went back and updated one of the delete cases to delete the permit attribute.

@kerry-meyer

File: playbooks/common_examples/sonic_bgp.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kerry-meyer) [reply playbooks/common_examples/sonic_bgp.yaml]: Verified.

File: playbooks/common_examples/sonic_bgp.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kerry-meyer) [reply playbooks/common_examples/sonic_bgp.yaml]: Verified.

File: playbooks/common_examples/sonic_bgp.yaml (hunk 3 above)
Comment (kerry-meyer) [reply playbooks/common_examples/sonic_bgp.yaml:263]: Thanks for explaining. I see the other example.

**Comment by kerry-meyer** [comment]
Thanks for providing the additional documentation updates and addressing all of the review comments.

The current version of the change set looks good.

Approved.

**Review by kerry-meyer — APPROVED** [review]
Approved.

**Pull request merged**